cairo/tests/ethereum/utils/test_numeric.py

```diff
@@ -81,3 +82,30 @@ def test_U256__eq__(self, cairo_run, a: U256, b: U256):
     @given(address=...)
     def test_U256_from_be_bytes20(self, cairo_run, address: Address):
         assert U256.from_be_bytes(address) == cairo_run("U256_from_be_bytes20", address)
+
+    @given(a=..., b=...)
+    def test_U256_le(self, cairo_run, a: U256, b: U256):
+        assert (a <= b) == cairo_run("U256_le", a, b)
+
+    @given(a=..., b=...)
+    def test_U256_add(self, cairo_run, a: U256, b: U256):
+        try:
+            a + b
+        except Exception as e:
+            with cairo_error(str(e.__class__.__name__)):
+                cairo_result = cairo_run("U256_add", a, b)
```

Comment:
```suggestion
            with strict_raises(type(e)):
                cairo_result = cairo_run("U256_add", a, b)
```

and in runner.py add a mechanism to throw the correct error based on the string: "Error message: OverflowError"